Docs/sphinx_documentation/source/External_Profiling_Tools.rst

```diff
@@ -279,3 +279,167 @@ generated figures. Some examples are shown here.
 
 .. [5]
    https://www.nersc.gov/users/software/performance-and-debugging-tools/ipm/
+
+NSight Systems
+==============
+
+The NSight Systems tool provides a high-level overview of a GPU code, displaying the kernel
+launches, API calls, NVTX regions and more in a timeline for a clear, visual picture of the
+overall runtime patterns.  It analyzes CUDA-based GPU codes and is available on Summit and
+Cori in a system module.
+
+NSight Systems provides a variety of profiling options.  This documentation will cover the
+most commonly used options for AMReX users to keep track of useful flags and analysis
+patterns. For the complete details of using NSight Systems, refer to the official documentation.
+
+Profile Analysis
+----------------
+
+The most common use case of NSight Systems for AMReX users is the creation of a qdstrm file
+that is viewed in the NSight Systems GUI, typically on a local workstation or machine.
+
+To generate a dqstrm file, run nsys with the ``-o`` option:
+
+.. highlight:: c++
+
+    nsys profile -o <file_name> ${EXE} ${INPUTS}
+
+AMReX's lambda-based launch system often makes these timelines difficult to parse, as the kernel
+are mangled and are difficult to decipher. AMReX's Tiny Profiler includes NVTX region markers,
+which can be used to mark each section of the NSight Systems timeline and.  To include AMReX's
+built-in Tiny Profiler NVTX regions in NSight Systems outputs, compile AMReX with 
+``TINY_PROFILE=TRUE`` and run nsys with ``-e NSYS_NVTX_PROFILER_REGISTER_ONLY=0``: 
```

Comment:
This registered string environment variable is only needed if you're trying to use the `--capture-range` functionality. You don't need it if you're just trying to collect a profile with NVTX regions.

Reply:
Just to be clear, I would need the environment variable when using `-c nvtx`, not when using `-c cudaProfilerApi`, correct?

Reply:
Correct.

Reply:
👍